oz/Guest.py

```diff
@@ -526,6 +526,11 @@ def _generate_xml(self, bootdev, installdev, kernel=None, initrd=None,
             install = oz.ozutil.lxml_subelement(devices, "disk", None, {'type':'file', 'device':installdev.devicetype})
             oz.ozutil.lxml_subelement(install, "source", None, {'file':installdev.path})
             oz.ozutil.lxml_subelement(install, "target", None, {'dev':installdev.bus})
+        # qemu-ga
+        channel = oz.ozutil.lxml_subelement(devices, "channel", None, {'type':'unix'})
+        source_path = "/var/lib/libvirt/qemu/%s.%s.sock" % ("org.qemu.guest_agent.0", self.tdl.name)
```

Comment:
I like the idea, but I have a question.  Will the hard-coded `/var/lib/libvirt/qemu` path work for non-root installations?

Reply:
Thanks clalancette!
1. libivrt is a daemon running with root permission
2. I checked this path on my environment(Centos 7.3), its belong to qemu:qemu(this really make sense, cause it used by qemu directly)